spec/fixtures/model/force/application/paper_app_domain.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
I manually created this json by filling out every field in a paper application on the frontend